memiavl/db.go

```diff
@@ -204,7 +209,7 @@ func (db *DB) RewriteSnapshotBackground() error {
 	wal := db.wal
 	go func() {
 		defer close(ch)
-		path, err := cloned.RewriteSnapshot()
+		version, err := cloned.RewriteSnapshot()
```

Comment:
`version` is just the `cloned.lastCommitInfo.Version`?

Reply:
it's return from RewriteSnapshot, CatchupWAL will update it

Reply:
The CatchupWAL is done on the newly loaded `mtree`, the `cloned` db is not updated.